### ISSUE TITLE
Bug: openy_gc_auth_custom_update_8009 throws error

### ISSUE THREAD

**Issue by froboy** [ymcatwincities/openy_gated_content#129]
https://github.com/ymcatwincities/openy_gated_content/blob/2d90ec5dd886224179f63d201541822b26e45342/modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.install#L155

When running `drush updb` I see:

```
File modules/contrib/openy_gated_content/modules/openy_gc_auth/modules/openy_gc_auth_custom/config/openy_gc_auth.provider.custom.yml does not exist.      [error]
File modules/contrib/openy_gated_content/modules/openy_gc_auth/modules/openy_gc_auth_custom/config/migrate_plus.migration.gc_auth_custom_users.yml does   [error]
not exist.
Performing openy_gc_auth_custom_update_8009  
```

`install` is missing from the `config_dir`.

**Comment by sarah-halby** [comment]
@anpolimus which release was this merged to? 1.3 or 1.4?

**Comment by sarah-halby** [comment]
@anpolimus was this included in 1.3 or 1.4?

**Comment by anpolimus** [comment]
@sarah-halby 1.4

**Comment by sarah-halby** [comment]
Thank you @anpolimus - closing this ticket. Merged in the 1.4 (May 2021) release.
